src/dotnet/UI.Blazor.App/Pages/HomePage.razor

```diff
@@ -31,9 +39,18 @@
 
     protected override async Task<Model> ComputeState(CancellationToken cancellationToken) {
         var account = await Accounts.Get(Session, CancellationToken.None).ConfigureAwait(false) ?? Account.Guest;
-        var chat = await RecentChatsUI.GetSelectedOrDefaultChat(cancellationToken).ConfigureAwait(false);
-        return new(account, chat?.Id ?? Symbol.Empty);
+        string link = "";
+        if (HistoryUI.IsInitialLocation) {
+            if (BrowserInfo.ScreenSize.Value.IsNarrow())
+                link = Links.ChatPage("");
```

Comment:
This seems weird... I.e. why there is a difference based on screen size?

Reply:
In narrow view address '/chat/' is used to display navbar only.
In wide view there is no sense to have this additional address.
I think we should discuss if it's needed to have the same behavior in both width sizes.

Reply:
Aha, I see now.